### ISSUE TITLE
How to fix corrupted file on git-lfs server?

### ISSUE THREAD

**Issue by PetrBodnar** [git-lfs/git-lfs#3364]
We are using git-lfs v.2.4.2 (as part of GitLab 11.2.1-ee) enabled.    
We found that we have corrupted object in git-lfs on server. When we run   

    sudo gitlab-rake gitlab:lfs:check VERBOSE=1 

, we have

    - 1401..1600: Failures: 1
      - LFS object: b675b00828707cd53dc5c130efce8a9d352e5efd8b9b29f06808810b6ebb5491: Checksum mismatch

Now when we try to do 
```git lfs pull``` in clones, we got  

    Expected OID b675b00828707cd53dc5c130efce8a9d352e5efd8b9b29f06808810b6ebb5491, got 2c6abc2c4d29bd1c7f0041160ea15ea2765df10194a6e9ef3e6e2139c11fc67f after 12427595 bytes written
    error: failed to fetch some objects from 'https://192.168.1.100/root/our_project.git/info/lfs'

We tried to change the related files and recommit them, but that didnt help. 
We cannot use new copies of our repo now, and we didnt found any info in Internet on such issues. Is there any way to fix this corruption?

**Comment by PetrBodnar** [comment]
Solved.

We found that sha256sum of contents of file b6/75/b00828707cd53dc5c130efce8a9d352e5efd8b9b29f06808810b6ebb5491 on server equals to 2c6abc2c4d29bd1c7f0041160ea15ea2765df10194a6e9ef3e6e2139c11fc67f (which seems to be incorrect), and contents of same file on repo clone is correct (equals to b675b00828707cd53dc5c130efce8a9d352e5efd8b9b29f06808810b6ebb5491).
We replaced contents of file b675b00828707cd53dc5c130efce8a9d352e5efd8b9b29f06808810b6ebb5491 on server with contents of this file from repo clone, and all is ok now.
